analysis/study_definition_delivery.py

```diff
@@ -94,6 +61,41 @@
             "incidence": 0.75,
         },
     ),
+            
+            
+    # fill missing ethnicity from SUS
+    ethnicity_6_sus = patients.with_ethnicity_from_sus(
+        returning="group_6",  
+        use_most_frequent_code=True,
+        return_expectations={
+            "category": {"ratios": {"1": 0.2, "2": 0.2, "3": 0.2, "4": 0.2, "5": 0.2}},
+            "incidence": 0.4,
+            },
+    ),
```

Comment:
I know this is new this week - I can't find documentation (apart from issue where decisions were made). This comment is to trigger documentation and maybe use as example

Reply:
Documentation is written in cohortextractor, just needs pulling into docs with a small update - I've prompted the devs to do this.